indico/modules/rb/client/js/common/rooms/RoomEditModal.jsx

```diff
@@ -8,951 +8,309 @@
-          type: 'input',
-          name: 'number',
-          label: Translate.string('Number'),
-          required: true,
-          inputArgs: {
-            fluid: true,
-          },
-        },
-      ],
-    },
-    {
-      type: 'formgroup',
-      key: 'coordinates',
-      content: [
-        {
-          type: 'input',
-          name: 'latitude',
-          label: Translate.string('Latitude'),
-          inputArgs: {
-            type: 'text',
-            fluid: true,
-          },
-          parse: v => p.number(v, false),
-          required: false,
-        },
-        {
-          type: 'input',
-          name: 'longitude',
-          label: Translate.string('Longitude'),
-          inputArgs: {
-            type: 'text',
-            fluid: true,
-          },
-          parse: v => p.number(v, false),
-          required: false,
-        },
-      ],
-    },
-    {
-      type: 'input',
-      name: 'surfaceArea',
-      label: Translate.string('Surface Area (m²)'),
-      inputArgs: {
-        type: 'number',
-        min: 0,
-      },
-      required: false,
-    },
-    {
-      type: 'input',
-      name: 'maxAdvanceDays',
-      label: Translate.string('Maximum advance time for bookings (days)'),
-      inputArgs: {
-        type: 'number',
-        min: 1,
-      },
-      required: false,
-    },
-    {
-      type: 'input',
-      name: 'bookingLimitDays',
-      label: Translate.string('Max duration of a booking (day)'),
-      inputArgs: {
-        type: 'number',
-        min: 1,
-      },
-      required: false,
-    },
-    {
-      type: 'header',
-      label: Translate.string('Options'),
-    },
-    {
-      type: 'checkbox',
-      name: 'isReservable',
-      label: Translate.string('Bookable'),
-    },
-    {
-      type: 'checkbox',
-      name: 'reservationsNeedConfirmation',
-      label: Translate.string('Require confirmation (pre-bookings)'),
-    },
-    {
-      type: 'checkbox',
-      name: 'notificationsEnabled',
-      label: Translate.string('Reminders enabled'),
-    },
-    {
-      type: 'checkbox',
-      name: 'endNotificationsEnabled',
-      label: Translate.string('Reminders of finishing bookings enabled'),
-    },
-    {
-      type: 'header',
-      label: Translate.string('Notifications'),
-    },
-    {
-      type: 'emails',
-      name: 'notificationEmails',
-      label: Translate.string('Notification emails'),
-    },
-    {
-      type: 'formgroup',
-      key: 'notifications',
-      dependsOn: 'notificationsEnabled',
-      label: Translate.string('How many days in advance booking reminders should be sent'),
-      content: [
-        {
-          type: 'input',
-          name: 'notificationBeforeDays',
-          label: Translate.string('Single/Daily'),
-          inputArgs: {
-            type: 'number',
-            min: 1,
-            max: 30,
-            fluid: true,
-          },
-          required: false,
-        },
+  const fetch = useCallback(async url => {
```

Comment:
I'd still prefer a different name since `fetch` is also the name of a builtin.

Reply:
Any suggestions?

Reply:
loadData, fetchData, ...